.gitignore

```diff
@@ -1,10 +1,16 @@
-node_modules
+# Compiler files
+cache/
+out/
+
+# Ignores development broadcast logs
+!/broadcast
+/broadcast/*
+/broadcast/*/31337/
+
+# Dotenv file
 .env
-coverage
-coverage.json
-typechain
-.DS_store
 
-#Hardhat files
-cache
-artifacts
```

Comment:
Also artifacts as well

Reply:
There are no artifacts. The project no longer uses hardhat